static/explorer.scss

```diff
@@ -372,7 +372,7 @@ pre.content.wrap * {
 
 .change-language {
     line-height: 14px;
-    min-width: 150px !important;
+    min-width: 200px !important;
```

Comment:
More space was required to fit the "Search for language..." placeholder text + it looked very cramped with only 150px. I'm not sure if this affects the rest of the layout though.

Reply:
Should be ok